### ISSUE TITLE
Spotify Connect support

### ISSUE THREAD

**Issue by awlx** [lukedirtwalker/cutespotify#25]
Hi there,

I know you already wrote on openrepos that there is no API for that. But maybe it can be implemented using this:
https://github.com/sashahilton00/spotify-connect-resources

Cheers

**Comment by lukedirtwalker** [comment]
Yeah I have seen this project. As I previously said I have no devices (to test etc.), but if anyone wants to do this this might be a good starting point.

**Comment by awlx** [comment]
You don't need any special devices. Just start the Spotify Client on your Desktop and the Android app will find it using Spotify Connect.

**Comment by jgueytat** [comment]
Also looking for Spotify Connect. I'm gonna have a look if I can't add this feature.
Just found this link : 
https://beta.developer.spotify.com/documentation/web-api/guides/using-connect-web-api/
